doc/advancedFeatures.rst

```diff
@@ -0,0 +1,43 @@
+Advanced Features
+=================
+.. Gradient Evaluation with Complex Step
+.. -------------------------------------
+
+.. Parallel Execution
+.. ------------------
+
+.. Storing Optimization History
+.. ----------------------------
+
+Hot start
+---------
+Hot start refers to the way optimizations are initialized.
+Suppose you run an optimization, and it was accidentally terminated prematurely, for example due to an iteration limit which was too low.
+If you restarted the optimization from the last iteration, you will likely incur a performance penalty, and end up taking far more steps than it would've taken if the original optimization was allowed to progress.
```

Comment:
Worth clarifying that this happens bc you restart "from scratch" with just a different set of initial DVs?
E.g. for some GB opts the first iterations are expensive bc you need to rebuild the Hessian? Also links with the following sentence

Reply:
Yeah that would work well. Would you mind doing some small edits?

Reply:
Sure, I will edit this by tomorrow the latest, thanks for having done most of the job already!